packages/typography/src/Link/Link/Link.tsx

```diff
@@ -127,9 +123,8 @@ const Link = InferredPolymorphic<BaseLinkProps, 'span'>(
         {...defaultAnchorProps}
         {...rest}
       >
-        <span className={cx(underlineStyles, underlineModeStyles[theme])}>
-          {children}
-        </span>
+        <span>{children}</span>
+
```

Comment:
do we need this empty space?

Reply:
its how the DOM is structured now - rm'ing it could break tests for folks 

something we could fix with test harnesses but wasn't sure it was worth the majro change to get this fix out